### PR TITLE
DOC: Add to FAQ

### PR DESCRIPTION
**Pull request by haydenrob**

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -3,3 +3,154 @@
...
+    my_material = RI([n, k])
+
+
+What EMA methods does *refellips* provide?
+------------------------------------------
+
+*refellips* offers the three main methods of effective medium approximations
+(EMA): linear, Maxwell-Garnett and Bruggeman. All EMA calculations performed
+in *refellips* are done based on two-component mixing, and done so using the
+complex dielectric function, and not refractive indicies and extinction
```
Comment (andyfaff) [inline docs/faq.rst:85]: indices

File: docs/faq.rst [hunk 2]
```diff
@@ -3,3 +3,154 @@
...
+If required, users can download their own dispersion curves from
+`refractiveindex.info`_ and load them into *refellips* using::
+
+    my_material = RI("my_dispersion.csv")
+
+The loaded file must contain at least two columns, assumed to be wavelength
+(in microns) and refractive index. If three columns are provided the third is
+loaded as the extinction coefficient.
+
+Alternatively, users have the option to specify Cauchy parameters (a,b,c) for their material::
```
Comment (andyfaff) [inline docs/faq.rst:69]: Can we have line length 80 in the file

**Review by andyfaff — APPROVED** [review]

File: docs/faq.rst (hunk 2 above)
Comment (haydenrob) [reply docs/faq.rst:69]: Not sure I follow

Comment (andyfaff) [reply docs/faq.rst:69]: We have a max line length of 80 characters in all the Python files. Could we have the same in RST please? (except equation blocks, etc)

**Pull request merged**